Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -0,0 +1,2860 @@
+commonfields:
+  id: BmcITSM
+  version: -1
+name: BmcITSM
+display: BMC Helix ITSM
+category: Utilities
+description: BMC Helix ITSM integration allows customers to manage service request, incident, change request, task, problem investigation and known error tickets.
+configuration:
+  - name: url
+    display: Server URL
+    required: true
+    defaultvalue: https://localhost:8008
+    type: 0
+    additionalinfo: "For Example: https://localhost:8008"
+  - name: credentials
+    display: Username
+    required: true
+    defaultvalue:
+    type: 9
+    additionalinfo: ""
+    displaypassword: Password
+    hiddenusername: false
+  - name: max_fetch
+    display: Maximum incidents per fetch.
+    required: false
+    defaultvalue: "50"
+    type: 0
+    additionalinfo: Default is 50. Maximum is 200.
+  - name: first_fetch
+    display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days).
+    required: false
+    defaultvalue: 7 days
+    type: 0
+    additionalinfo: ""
+  - name: ticket_type
+    display: Ticket Type
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The type of the tickets to fetch.
+    options:
+      - service request
+      - incident
+      - task
+      - change request
+      - problem investigation
+      - known error
+      - All
+  - name: ticket_status
+    display: Ticket Status
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The status of the tickets to fetch. Since Each ticket type has it's own unique status pool, consider to select only those who match to the selected ticket types.
```

Comment:
I recommend "The status of the tickets to fetch. Since each ticket type has its own unique status pool, consider selecting only those who match the selected ticket types."